app/controllers/robots_txts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RobotsTxtsController < ApplicationController
+  skip_before_action :require_basic_auth
+
+  def show
+    if disallow_all_crawlers?
+      render template: :disallow_all, layout: false, content_type: 'text/plain'
+    else
+      render body: nil, status: :not_found
+    end
+  end
+
+  private
+
+  def disallow_all_crawlers?
+    ENV['DISALLOW_ALL_WEB_CRAWLERS'].present?
```

Comment:
```suggestion
    ENV['DENY_ALL_WEB_CRAWLERS'].present?
```

are we going to set a default value for this in the editor which injects it into the runner container? if not then we should set a default value here.